client/src/rpc_client.rs

```diff
@@ -728,6 +729,14 @@ impl RpcClient {
             .ok_or_else(|| RpcError::ForUser(format!("AccountNotFound: pubkey={}", pubkey)).into())
     }
 
+    /// Note that `get_account` returns `Err(..)` if the account does not exist whereas
+    /// `get_account_with_commitment` returns `Ok(None)` if the account does not exist.
+    pub fn get_account_no_data(&self, pubkey: &Pubkey) -> ClientResult<AccountNoData> {
```

Comment:
@CriesofCarrots @jackcmay : @carllin suggested I call this type of rpc change out to you two in case we needed to do something different here.

Reply:
Thanks for the heads up! Is there anything in particular in the client or rpc tooling on which you have questions?

Reply:
@CriesofCarrots: @carllin and I expressed some lack of knowledge in the rpc area. @sakridge raised rpc as a concern early on with this refactoring and rename. So, we need an expert to make sure we can add methods here and things will work ok and to see if we missed something.